src/plugins/OEmbed/Components.tsx

```diff
@@ -71,38 +78,42 @@ export const OEmbed = (
   if ("html" in props) {
     const processedHtml = preprocessHtml(props.html);
     return (
-      <Article
-        data-src={props.attributes.src}
-        data-spoilers={props.attributes.spoilers}
-        data-width={props.attributes.width}
-        data-height={props.attributes.height}
-        data-source={getWebsiteNameFromUrl(props.meta.canonical)}
-        data-loaded={props.loaded}
-        dangerouslySetInnerHTML={{ __html: processedHtml }}
-        ref={onAttachNode}
-      />
+      <BlockBaseComponent {...props} className="embed" enclosingTag={"article"}>
+        <div
+          className={wrapperClass}
+          dangerouslySetInnerHTML={{ __html: processedHtml }}
+          data-source={getWebsiteNameFromUrl(props.meta.canonical)}
```

Comment:
i would say that src in place of source is fairly standard for HTML, so i wouldn't change it

Reply:
This isn't changing data-src (since src is an plugin attribute it automatically gets turned into a data attribute), this is just the data-source you had at line 79.

It's possible we should rename it though so that there isn't both a data-src and a data-source

Reply:
Oh good point. What about `data-site`? not in love with it, but better than source.

Reply:
I'll do `data-site` for now. Could also do `data-origin` or `data-hostname`